src/giskardpy/python_interface.py

```diff
@@ -282,6 +358,27 @@ def grasp_bar(self, root_link, tip_link, tip_grasp_axis, bar_center, bar_axis, b
                            max_angular_velocity=max_angular_velocity,
                            weight=weight)
 
+    def set_open_goal(self, tip_link, object_link_name, angle_goal,
+             weight=WEIGHT_ABOVE_CA):
+        """
```

Comment:
Choose a better name and don't hard code ```iai_kitchen```

Reply:
I have renamed it  to set_pull_door_goal. I hope it is an appropriate name.